packages/fluentui/react-northstar/src/components/Chat/ChatMessage.tsx

```diff
@@ -335,7 +335,7 @@ export const ChatMessage = (React.forwardRef<HTMLDivElement, ChatMessageProps>((
 
   const popperRef = React.useRef<PopperRefHandle>();
   const { targetRef: actionsMenuTargetRef, containerRef: actionsMenuRef } = usePopper({
-    align: 'end',
+    align: context.rtl ? 'start' : 'end',
```

Comment:
`usePopper` already handles `rtl`:

https://github.com/microsoft/fluentui/blob/a9c7195062b2ec7e6667d19783521e2ec99292c2/packages/fluentui/react-northstar/src/utils/positioner/positioningHelper.ts#L22-L23

The problem there is that `rtl` is not passed to `usePopper()` itself:

```suggestion
    align: 'end',
    rtl: context.rtl,
```